email/emails/registration-confirm.tsx

```diff
@@ -40,9 +41,18 @@ export default function RegistrationConfirmEmail() {
         />
       </Text>
       <Text>
+        <If condition="${isSignup}">
```

Comment:
Not sure using the same template for registration and email changes is advisable (given the advanced templating system now); subject of the email might be ambiguous and it limits the extensibility of the registration email (e.g. adding additional sections such as getting started guides and what not).

The wording of the `confirm-email-text` is a bit confusing too; people aren't "starting to use" Tolgee but rather "changing their email" and I think the email body should reflect that.

Related to the 1st point email change confirmation (and other important emails) should perhaps contain warnings and advise caution regarding phishing campaigns and what to do in case of suspicious activity (use MFA, change passwords, etc).